Autopilot/TelemetryManager/Inc/telemetryStateClasses.hpp

```diff
@@ -33,6 +33,7 @@ class obtainDataMode: public telemetryState
         void execute(telemetryManager* telemetryMgr);
         void exit(telemetryManager* telemetryMgr){(void) telemetryMgr;}
         static telemetryState& getInstance();
+	static XBEE& getInstance();
```

Comment:
You should not be making a method like this here. Since the XBEE class is a singleton, if a class wants to use an XBEE, it should just call the getInstance() method in teh XBEE class. 

If this is a result of confusion from my previous comment, this is what I meant :))

```cpp
class obtainDataMode: public telemetryState{    
public:        
  void enter(telemetryManager* telemetryMgr){(void) telemetryMgr;}       
  void execute(telemetryManager* telemetryMgr);       
  void exit(telemetryManager* telemetryMgr){(void) telemetryMgr;}        
  static telemetryState& getInstance();

private:        
  obtainDataMode(){ ZPXbee = XBEE::getInstance(); }        
  obtainDataMode(const obtainDataMode& other);        
  obtainDataMode& operator =(const obtainDataMode& other);        
  XBEE ZPXbee;
};
```